nex/nex.go

```diff
@@ -71,6 +71,7 @@ func init() {
 	ncli.Flag("logjson", "Log JSON").Default("false").Envar("NEX_LOGJSON").UnNegatableBoolVar(&Opts.LogJSON)
 	ncli.Flag("context", "Configuration context").Envar("NATS_CONTEXT").PlaceHolder("NAME").StringVar(&Opts.ConfigurationContext)
 	ncli.Flag("no-context", "Disable NATS context discovery").UnNegatableBoolVar(&Opts.SkipContexts)
+	ncli.Flag("conn-name", "Name of NATS connection").StringVar(&Opts.ConnectionName)
```

Comment:
If this flag is left off, should we use a reasonable default, like `nex-vXX` so the default behavior is to identify nex and its version?

Reply:
done!